.github/workflows/CI.yml

```diff
@@ -44,10 +44,27 @@ jobs:
 
       - name: Running Tests
         run: bundle exec rake spec
-
+  danger:
+    runs-on: ubuntu-latest
+    permissions:
```

Comment:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token#changing-the-permissions-in-a-forked-repository

> You can use the permissions key to add and remove read permissions for forked repositories, but typically you can't grant write access.